code/modules/client/preferences.dm

```diff
@@ -55,6 +62,8 @@ var/list/preferences_datums = list()
 	var/hair_color = "000"				//Hair color
 	var/facial_hair_style = "Shaved"	//Face hair type
 	var/facial_hair_color = "000"		//Facial hair color
+	var/beard_level_enabled = FALSE		//Enables beard level

```

Comment:
I dream of a world where beards are respected, where only the most robust or cowardly can have a knee-lengther